djedi/admin/api.py

```diff
@@ -193,11 +193,17 @@ def get(self, request, uri):
         try:
             uri = self.decode_uri(uri)
             uri = URI(uri)
-            plugins.resolve(uri)
+            plugin = plugins.resolve(uri)
+            plugin_context = self.get_context_data(uri=uri)
+
+            # TODO: Remove has_attr check when cio has default plugin context
+            if hasattr(plugin, 'get_context'):
```

Comment:
I think we should prefer using `except AttributeError` instead of `hasattr`. 

Some more details: https://hynek.me/articles/hasattr/

Reply:
Makes sense :+1: